blog/2024/10-26-jellyfin-release-10.10.0/index.mdx

```diff
@@ -0,0 +1,72 @@
+
+This major release brings several new features, improvements, and bugfixes to improve your Jellyfin experience. With our faster release cadence between 10.9.0 and 10.10.0, this release should be far less daunting, so please read on for a quick peek at what's new and some important-to-know breaking changes!
+
+You may upgrade your Jellyfin instances at any time now. For those who were running Unstable builds for testing, we thank you immensely, and you may now switch back to the Stable repository and forcibly reinstall/repull the latest version. As always, **ensure you back up your Jellyfin data and configuration directories before upgrading**. With a major release, it's possible you will hit a bug and want to revert, and to do so, you will need to restore from a backup.
+
+Happy watching!
+
+\- Joshua
+
+{/* truncate */}
+
+## Breaking Changes
+
+* Most clients should continue to work as-is without any issues or any forced upgrades, though this may change in the future. The sole exception is Jellyfin4Kodi, which currently has issues due to point 5 below, though a fix is hopefully forthcoming shortly.
+
+* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate any Raspberry Pi-specific functionality in general for releases past 10.10.0. Raspberry Pi has always been a tricky platform for us, as it's popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine and the aging of the RPi4, we are taking this action now to ensure it's widely known.  If you are running Jellyfin on old RPi hardware, now is the time to start [looking at an upgrade](https://jellyfin.org/docs/general/administration/hardware-selection). See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).
+
+* We now use the system temporary storage engine (e.g. `/tmp` on Linux) for storing short-lived transient temporary files, to allow us to leverage temporary ramdisks and the like and avoid cluttering potentially slow storage. This may cause issues if you specifically depended on the previous behaviour. Note that this does not include transcoding temporary files. See [PR #12226](https://github.com/jellyfin/jellyfin/pull/12226).
+
+* The server will now refuse to start if `ffmpeg` cannot be found, is an incorrect version, or does not function properly (missing extensions, etc.). With how critical `ffmpeg` is to Jellyfin, this has become very important to avoid mis-reported issues. This can be explicitly bypassed if needed. See [PR #12463](https://github.com/jellyfin/jellyfin/pull/12463).
+
+* Network paths in libraries have been fully removed and will no longer work. This functionality has been deprecated for a long time, and most of it was removed in 10.9.0, but this removes the remainder. See [PR #12446](https://github.com/jellyfin/jellyfin/pull/12446).
```

Comment:
Should mention that (third-party) clients relying on this feature can easily implement this feature themself.